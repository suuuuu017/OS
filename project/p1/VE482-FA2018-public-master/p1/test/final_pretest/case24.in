WRITELN /bin/ls &
WRITELN sleep 0.2
WRITELN sleep 0.2 &
WRITELN jobs
WRITELN sleep 0.4
WRITELN exit
