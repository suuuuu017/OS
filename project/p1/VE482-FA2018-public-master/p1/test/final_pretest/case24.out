mumsh $ [1] /bin/ls &
mumsh $ driver
mumsh
mumsh_memory_check
mumsh $ [2] sleep 0.2 &
mumsh $ [1] done /bin/ls &
[2] running sleep 0.2 &
mumsh $ mumsh $ exit
